src/players/twistmoe/twistmoe.ts

```diff
@@ -12,18 +12,22 @@ export class Twistmoe extends BasePlayer {
   }
 
   getVideoContainer(): HTMLElement | null {
-    return super.getContainerHelper(metadata.videoContainerSelectorString);
+    return super.getContainerHelper(
+      this.metadata.selectorStrings.default!.videoContainerSelectorString
```

Comment:
What's does the `!` in `default!.videoContainerSelectorString` actually do :?

Reply:
It tells the compiler that even though <variable> can be null, I am sure that it is not null when we try to access it